lib/rubocop/cop/layout/empty_lines_around_exception_handling_keywords.rb

```diff
@@ -72,7 +72,7 @@ def on_def(node)
         alias on_numblock on_def
 
         def on_kwbegin(node)
-          body, = *node
+          body, = *node # rubocop:disable InternalAffairs/NodeDestructuring
           check_body(body, node.loc.line)
```

Comment:
My previous suggestion to use `KeywordBeginNode.body` won't actually work without heavy refactoring because this cop looks at the rescue and ensure too. I'd just change it like so now:

```suggestion
          check_body(node.children.first, node.loc.line)
```